schedule-replicas-by-topology-labels.md

```diff
@@ -240,7 +240,7 @@ Then, assume that the number of cluster replicas is 5 (`max-replicas=5`). Becaus
 
```

Comment:
(non-related, but needs fixing) Also on line ... it is confusing to read about the configuration:
"In the following example, a two-layer topology of `zone/host` is defined. The TiKV nodes of the cluster are distributed among three zones, z1, z2, and z3, with each zone having four hosts, h1, h2, h3, and h4. In z1, four TiKV instances are deployed on two hosts, `tikv-1` and `tikv-2` on h1, and `tikv-3` and `tikv-4` on h2. Two TiFlash instances are deployed on the other two hosts, `tiflash-1` on h3 and `tiflash-2` on h4. In z2 and z3, two TiKV instances are deployed on two hosts, and two TiFlash instances are deployed on the other two hosts. In the following example, `tikv-n` represents the IP address of the `n`th TiKV node, and `tiflash-n` represents the IP address of the `n`th TiFlash node."

tikv-1..tikv-4 are four (virtual) machines, with each own IP address, so the statement:
"In z1, four TiKV instances are deployed on two hosts, `tikv-1` and `tikv-2` on h1, and `tikv-3` and `tikv-4` on h2."
is not correct (or at least very confusing) and are mixing tikv node names with machine names, take the example:
```
tikv_servers:
# z1
  - host: tikv-1
    port：20160
    config:
      server.labels:
        zone: z1
        host: h1
   - host: tikv-1
     port：20161
    config:
      server.labels:
        zone: z1
        host: h1
  - host: tikv-2
    port：20160
    config:
      server.labels:
        zone: z1
        host: h2
  - host: tikv-2
    port：20161
    config:
      server.labels:
        zone: z1
        host: h2
```
Here `- host: tikv-1` is the host machine (VM) not the node name, which may be given two names 'tikv-1' and 'tikv-2' which will make it very confusing, since node 'tikv-2' will be running on machine 'tikv-1', together with node 'tikv-1' but on a different port. It is later also given a different name in the label 'config . server.labels . host': 'h1' instead of using the same machine name also for the label.

I suggest using 'tikv-host-machine-n' as the host name, as well as in config/server.labels/host. That would make less conflict with generated node names, such as tikv-1.

Reply:
Updated. PTAL.